packages/next-app/src/app/pool/[poolId]/MetadataAttribute.tsx

```diff
@@ -30,91 +75,114 @@ const Header = () => {
   return (
     <thead>
       <tr>
-        {cols.map(({ node, className }) => (
-          <Th className={className}>{node}</Th>
+        {cols.map(({ node, className }, index) => (
+          <Th key={index} className={className}>
+            {node}
+          </Th>
         ))}
       </tr>
     </thead>
   );
 };
 
-const Td = ({ className, children }: CellProps) => (
-  <td
-    className={cn(
-      "whitespace-nowrap py-4 px-3 text-sm text-gray-300",
-      className
-    )}
-  >
-    {children}
-  </td>
-);
-
-const cellData: { node: React.ReactNode; className?: string }[][] = [
-  [
-    { node: "Title" },
-    { node: "text" },
-    { node: "An image URL of the pool" },
-    { node: "This pool contains many tokens..." },
-  ],
-];
-
-function Row({ data }: { data: typeof cellData }) {
-  const defaultActions = [
-    {
-      className:
-        "relative whitespace-nowrap py-4 pl-3 pr-4 text-right text-sm font-medium sm:pr-0",
-      node: (
-        <a href="#" className="text-indigo-400 hover:text-indigo-300">
-          Edit
-        </a>
-      ),
-    },
-  ];
+const Td = ({ className, children }: CellProps) => {
+  return (
+    <td className={cn("whitespace py-4 px-3 text-sm text-gray-300", className)}>
+      {children}
+    </td>
+  );
+};
 
-  const rows = data.map((row) => [...row, ...defaultActions]);
+const AttributeLink = ({ data }: { data: (typeof cellData)[1] }) => {
+  switch (data.type) {
+    case "URL":
+      return (
+        <button>
+          <ArrowTopRightIcon className="text-white" />
+        </button>
+      );
+    case "image":
+      return (
+        <button>
+          <ImageDialog src={data.value as string} />
+        </button>
```

Comment:
my personal take is that images should also open on a new tab, like links... but we don't need to spend any time here for now